odxtools/outputparam.py

```diff
@@ -28,23 +24,12 @@ def __post_init__(self) -> None:
     @staticmethod
     def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment]) -> "OutputParam":
 
-        odx_id = odxrequire(OdxLinkId.from_et(et_element, doc_frags))
-        short_name = odxrequire(et_element.findtext("SHORT-NAME"))
-        long_name = et_element.findtext("LONG-NAME")
-        description = create_description_from_et(et_element.find("DESC"))
+        kwargs = IdentifiableElement.get_kwargs(et_element, doc_frags)
         dop_base_ref = odxrequire(OdxLinkRef.from_et(et_element.find("DOP-BASE-REF"), doc_frags))
         semantic = et_element.get("SEMANTIC")
         oid = et_element.get("OID")
```

Comment:
@andlaus I noticed that most of the time (there are like only two exceptions), when element have an ID, it also have an optional OID attribute

Refactoring of OID could be done in different PR